documentation/BucketInfoModelVersion.md

```diff
@@ -245,4 +245,16 @@ For capacity-enabled buckets, contains the following data:
 
 ### Usage
 
-Used to store bucket tagging
\ No newline at end of file
+Used to store bucket tagging
+
+## Model version 17
+
+### Properties Added
+
+```javascript
+this._quota = quota || 0;
```

Comment:
Could be athe same key as Vault (quotaMax) if we want to be consistent

Reply:
done here : https://github.com/scality/Arsenal/pull/2234/commits/a5541f28bc6eb5fe46e0ac5e658a2be100057e15#diff-ffb3da4096d0bcb75ae13838bd7090485735b50eebf5e1ea1535641969bc9e07R255